components/ui/Footer.js

```diff
@@ -15,42 +40,48 @@ export const Footer = () => {
         </p>
         <p>&copy; 2024 Galactica. All rights reserved.</p>
       </div>
-      {/* TASK - React 1 week 2 */}
-      {/* Create a new List for the Pages */}
-      {/* We need to use the <Link /> component here */}
-      {/* <div className={styles.pages}>
-        <h3>Pages</h3>
-        <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
 
-      <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
-        <ul className={styles.footerList}>
-          <li>
-            <a href="https://www.linkedin.com">Linkedin</a>
-          </li>
+      <div className={styles.pages}>
+        <h3>Pages</h3>
+        <ul className={styles.link}>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            <Link href="/">Home</Link>
           </li>
           <li>
-            <a href="https://instagram.com">Instagram</a>
+            <Link href="/about_us" target="_blank">
+              About Us
+            </Link>
           </li>
           <li>
-            <a href="https://tiktok.com">Tiktok</a>
+            <Link href="/destination" target="_blank">
+              Destination
+            </Link>
           </li>
           <li>
-            <a href="https://google.com">On the streets at night</a>
+            <Link href="/nasa_collaboration" target="_blank">
+              NASA_ Collaboration
+            </Link>
           </li>
+        </ul>
+      </div>
+      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
 
-          {/* TASK - React 1 week 2 */}
-          {/* Create a <SocialMediaItem /> component and replace all of the list items! */}
-          {/* it should accept the following props */}
-          {/* url, title, icon */}
-          {/* For the icons, you can download 1-2 social media icons for testing and put it in the /public/socialmedia/ folder */}
+      <div className={styles.footerLinks}>
+        <h3>Follow us</h3>
+        <ul className={styles.footerList}>
+          {SocialMediaList.map((el) => {
+            return (
+              <>
```

Comment:
this can be removed and you can just use the `<li>` as the main level html component

Reply:
noted and corrected